core/static/js/opencanada.js

```diff
@@ -138,7 +145,19 @@ function toggleHeading(windowWidth){
 }
 
 $(window).resize(function(){
-    initForWindow()
+    var windowHeight = $(window).height();
```

Comment:
Perhaps refactor this chunk of code into a function (something like setupHTMLClick or something more descriptive), since the same code is used in multiple places.  It should make it easier to maintain in the future.